treeherder/webapp/api/push.py

```diff
@@ -6,21 +6,16 @@
 from rest_framework import viewsets
 from rest_framework.decorators import action
 from rest_framework.response import Response
-from rest_framework.status import (HTTP_400_BAD_REQUEST,
```

Comment:
MOST of these changes were done by the new `black` plugin when I did my commit.  Just reformatting.